ArmTemplates/README.md

```diff
@@ -0,0 +1,33 @@
+# ARM Templates
+
+This folder contains ARM deployments that can be used to stand up a specific resource and can be used in the same way as nested templated.

```

Comment:
Aren't they linked templates if you're calling from a master file?  https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-linked-templates

Reply:
That's exactly what they are. I had used the resource type - fixed and added link you provided